src/08_comprehensions.py

```diff
@@ -11,23 +11,25 @@
 # Write a list comprehension to produce the array [1, 2, 3, 4, 5]
 
```

Comment:
comprehensions are going to be one-liners , i.e: y = [i+1 for i in range(5)] , you have loops